Sources/SwiftProtobuf/JSONEncodingOptions.swift

```diff
@@ -27,5 +27,19 @@ public struct JSONEncodingOptions: Sendable {
   /// By default they are converted to JSON(lowerCamelCase) names.
   public var preserveProtoFieldNames: Bool = false
 
+  /// Whether to use deterministic ordering when serializing.
+  ///
+  /// Note that the deterministic serialization is NOT canonical across languages. 
+  /// It is not guaranteed to remain stable over time. It is unstable across 
```

Comment:
s/not/NOT/ (just to emphasis things the same as the previous sentence)

Reply:
Updated